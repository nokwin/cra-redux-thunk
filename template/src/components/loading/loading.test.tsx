import React from "react";
import renderer from "react-test-renderer";

import { Loading } from "./loading.component";

describe("Loading component", () => {
  it("should render loading component", () => {
    const tree = renderer.create(<Loading />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
