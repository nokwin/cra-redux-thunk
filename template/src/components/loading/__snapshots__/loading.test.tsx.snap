// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should render loading component 1`] = `
<h1>
  Loading...
</h1>
`;
